svelte/index.js

```diff
@@ -21,6 +22,10 @@ module.exports = {
           typescript: '@typescript-eslint/parser',
         },
       },
+      rules: {
```

Comment:
Please do also disable `no-undef-init` so we can define optional props in svelte components